test/compress/dead-code.js

```diff
@@ -256,3 +256,16 @@ try_catch_finally: {
         "1",
     ]
 }
+
+accessor: {
+    options = {
+        side_effects: true,
+    }
+    input: {
+        ({
+            get a() {},
+            b: 1
+        });
+    }
+    expect: {}
```

Comment:
Pardon my brevity - setter added in https://github.com/mishoo/UglifyJS2/pull/1926/commits/3562e864ea0e571ec6306312b482b4016ed88e73